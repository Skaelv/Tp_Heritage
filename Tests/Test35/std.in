LOAD Tests/Test35/Test35 
UNDO
EXIT
