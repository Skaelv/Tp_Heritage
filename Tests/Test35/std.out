 # Entrez une commande
LOAD Tests/Test35/Test35 
UNDO
EXITOK
# Loading new empty model
# New Object : C1
# New Object : C2
# New Object : R1
OK
# New Object : PL1
# New Object : OA1
# Model loaded
 # Entrez une commande
OK
 # Entrez une commande

